mlflow/data.py

```diff
@@ -20,7 +20,7 @@ class DownloadException(Exception):
 
 def _fetch_dbfs(uri, local_path):
     print("=== Downloading DBFS file %s to local path %s ===" % (uri, os.path.abspath(local_path)))
-    process.exec_cmd(cmd=["databricks", "fs", "cp", "--overwrite", uri, local_path])
+    process.exec_cmd(cmd=["databricks", "fs", "cp", "-r", uri, local_path])
```

Comment:
orthogonal, but adds support for recursively downloading. Overwrite seems scary; I don't think we were using that behavior anyway.